pages/osx/open.md

```diff
@@ -16,7 +16,7 @@
 
 - Reveal a file in finder:
 
-`open -R /path/dir/{{file}}`
+`open -R {{/path/to/file}}`
```

Comment:
Maybe remove the leading slash here? I believe the convention is to omit it, but not sure if you aimed at fixing that too.